### PR TITLE
navbar & improve sidenav & responsive design

### PR DESCRIPTION
**Pull request by huadream**
I didn't add some translations for Russian because I don't know Russian.
There is also something to do for mobile devices layout.

desktop
image

tablets
image
image

phone
image
image
image

other instructions
https://github.com/bwsw/cloudstack-ui/pull/1668#issuecomment-509276400

### CODE REVIEW

File: src/i18n/ru.json [hunk 1]
```diff
@@ -340,8 +340,9 @@
     "PASSWORD_REQUIRED": "Необходимо заполнить пароль"
   },
   "NAVIGATION_SIDEBAR": {
-    "WELCOME": "Здравствуйте, {{ name }}",
+    "WELCOME": "Здравствуйте, {{ firstname }}·{{ lastname }}",
```
Comment (dron8552) [inline src/i18n/ru.json]: should be the space symbol.

**Comment by dron8552** [comment]
We think that it will be better to constantly use 'Search...' for the search box placeholder. It is intuitive that the search will be for the current context.

**Comment by dron8552** [comment]
run tests

**Comment by dron8552** [comment]
run tests

File: src/app/home/home.component.ts [hunk 2]
```diff
@@ -45,24 +59,31 @@ export class HomeComponent extends WithUnsubscribe() implements OnInit {
           }),
         );
       });
-  }
 
-  public openAppNav() {
-    this.store.dispatch(new layoutStore.actions.OpenAppNav());
+    // When router changes and sideNavMode is 'over', hide the sideNav.
+    this.currentSubroute$.subscribe(() => {
+      if (this.sideNavMode === 'over' && this.sideNavOpen === true) {
```
Comment (dron8552) [inline src/app/home/home.component.ts]: Please, fix lint error: `ERROR: /src/app/home/home.component.ts[65, 42]: This expression is unnecessarily compared to a boolean. Just use it directly.`
Also, always run lint/test/e2e before commit.

Comment (ershovaelena) [reply src/app/home/home.component.ts]: @huadream Hello, we are ready to give your PR to our QA, please fix lint error described above

Comment (huadream) [reply src/app/home/home.component.ts]: I am very sorry that I get out for several days and e2e test didn't fix.

**Comment by dron8552** [comment]
run tests

**Comment by dron8552** [comment]
run tests

**Pull request merged**